package.json

```diff
@@ -21,9 +21,7 @@
     "@vuepress/plugin-medium-zoom": "^1.8.2",
     "markdown-it": "^12.2.0",
     "markdown-it-footnote": "^3.0.3",
-    "markdown-it-multimd-table": "^4.1.0",
-    "npm-check-updates": "^11.8.3",
-    "vuepress": "^1.8.2",
+    "vuepress": "^1.5.3",
```

Comment:
Please revert these changes, take note that you are reverting to older dependency versions for some reason.